docs(main): update source config doc (#3754)
web-infra-dev/modern.js: packages/document/main-doc/docs/en/configure/app/source/enable-async-entry.mdx

        └─ index.html
```

The contents of `index.js` are as follows:

```js
import('./bootstrap.jsx');
```

At this point, you can consume any remote module in the current page.

:::info

> Modern.js does not wrap webpack's ModuleFederationPlugin. Please configure the ModuleFederationPlugin yourself through [tools.webpackChain](/configure/app/tools/webpack-chain).

Reviewer: ```suggestion
Modern.js does not have webpack's ModuleFederationPlugin plugin built in. Please configure the ModuleFederationPlugin yourself via [tools.webpackChain](/configure/app/tools/webpack-chain).
```
